problemsets-mat223/modules/module5.tex

```diff
@@ -103,7 +125,22 @@
 \begin{example}
 	Find the component of $\vec a=\mat{1\\2}$ in the direction of $\vec b=\mat{1\\1}$.
 
-	XXX Finish
+    Let $\Vcomp_{\vec b}\vec a=\mat{x\\y}$ for some unknown $x,y\in\R$. 
+    
+    By definition, $\vec a - \Vcomp_{\vec b}\vec a$ is orthogonal to $\vec b$. That is,
+    \begin{equation}
+    \label{e1}
+        (\vec a-\Vcomp_{\vec b}\vec a)\cdot\vec b=0. \implies \mat{1-x\\2-y}\cdot\mat{1\\1} = 0 \implies x+y=3.
+    \end{equation}
+    Also from the definition, $\Vcomp_{\vec b}\vec a$ is the vector in the direction of $\vec b$. Thus, for some $k\in\R$, we have
+    \begin{equation}
+    \label{e2}
+        \Vcomp_{\vec b}\vec a = k\vec b \implies\mat{x\\y}=\mat{k\\k}\implies x-y=0.
```

Comment:
See above

Reply:
For equation references, use the naming convetion `EQxxx` where `xxx` is the name you want.